jsk_robot_common/jsk_robot_startup/lifelog/tweet_client.l

```diff
@@ -7,6 +7,10 @@
 (load "package://pr2eus/speak.l")
 
 (defun tweet-string (twit-str &key (warning-time) (with-image) (image-wait 30) (speak t))
+  (if (null (ros::get-num-subscribers "/tweet"))
+    (ros::advertise "/tweet" std_msgs::String 1))
+  (if (null (ros::get-num-subscribers "/tweet_image_server/tweet/goal"))
+    (ros::advertise "/tweet_image_server/tweet/goal" rostwitter::TweetActionGoal 1))
```

Comment:
Almost same as above.

Reply:
the origin/master branch does not use `rostwitter` message, so I removed this change